src/sourabh/testCase13/TestCase13Prop.java

```diff
@@ -0,0 +1,86 @@
+/*Test Case - 13 [18-06-2021]
+Program 1: count number of employee under each manager, verify count against properties file.
+output : 20209 -> 3
+         20205 -> 4
+ */
+
+package sourabh.testCase13;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Properties;
+import java.util.Set;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class TestCase13Prop {
+	static HashMap<String, Integer> getEmployeeCountEachManagerMap(WebDriver driver){
```

Comment:
If you create driver at instance level so we not require to pass reference of driver in every method